src/pybind/mgr/dashboard/frontend/src/app/shared/components/usage-bar/usage-bar.component.html

```diff
@@ -17,11 +17,15 @@
   <div class="progress-bar bg-info"
        [ngClass]="{'bg-warning': usedPercentage/100 >= warningThreshold, 'bg-danger': usedPercentage/100 >= errorThreshold}"
        role="progressbar"
+       [attr.aria-label]="title"
+       i18n-aria-label
```

Comment:
This is exactly the same as `aria-label="{ title }", which is not very useful for a translator (a translator could still work when the dynamic text has some context, like `"The title of this is { title }"`). What it needs to be translated is where the static text is set. Like the `usage` initialization.